src/v3/src/types/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = {
+  rules: {
+    'import/no-cycle': 'off',
```

Comment:
How difficult is it to resolve this? We will want to eventually untangle the types to avoid dependency cycles

Reply:
I don't really feel it matters for types, that why I added the extension file under types dir.

according to how to avoid this lint warning, we will need to put types into one giant file, which makes me feel type category weigh more than lint rule for this scenario

Reply:
That seems reasonable - I even found this issue `https://github.com/import-js/eslint-plugin-import/issues/1453` that makes a similar argument. I think we had trouble with using the `import type` syntax but at some point we can try migrating to it when we have time. Seems like that syntax solves the error without needing the lint ignore.